### DISCUSSION TITLE
can not connect after reboot. arm64 version 1.32

### DISCUSSION THREAD

**Discussion by clickerpw** [rustdesk/rustdesk#9800] [General]
### Bug Description

i have installed arm64 version on tv box with ubuntu from [https://github.com/devmfc/debian-on-amlogic](url)
after reboot i see  that my tvbox is online but i can not connect to rustdesk. connection refused.
after i login to tvbox via vnc. lunch rustdesk  from shortcut and press stop service in settings application window. close this window and press start service in main window. after this i can connect to rustdesk. can you help me,  that i need check 

### How to Reproduce

after reboot i see  that my tvbox is online but i can not connect to rustdesk

### Expected Behavior

connect after reboot

### Operating system(s) on local side and remote side

mac os-> arm64

### RustDesk Version(s) on local side and remote side

1.3.2

### Screenshots

![Снимок экрана 2024-10-31 в 21 53 07](https://github.com/user-attachments/assets/38224b77-76ff-4912-bc91-288fdb84b716)
![Снимок экрана 2024-10-31 в 21 53 32](https://github.com/user-attachments/assets/a13dedb3-e32e-4757-8595-7456894b65fd)
![Снимок экрана 2024-10-31 в 21 53 50](https://github.com/user-attachments/assets/2ff0d1c3-0d15-40f7-a276-e3cecd0a0212)

**Comment by 21pages** [comment]
I can't reproduce with arm64 ubuntu virtual machine, can you provide the logs on ubuntu? https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs. What does the message box display when `connection refused` appears?

**Reply by clickerpw** [reply]
> I can't reproduce with arm64 ubuntu virtual machine, can you provide the logs on ubuntu? https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs. What does the message box display when `connection refused` appears?
![Снимок экрана 2024-11-02 в 21 36 39](https://github.com/user-attachments/assets/c6399cee-ae15-4174-9a82-33ffe8feb00a)

**Reply by clickerpw** [reply]
I made a video with error connect demonstration and logs
[https://youtu.be/SDHKjeOGvhQ](https://youtu.be/SDHKjeOGvhQ)
after login to xrdp , rustdesk connection is established

**Comment by xlionjuan** [comment]
recommend to create a user with sudo permissions, login to DE with root is bad ideas....

**Reply by clickerpw** [reply]
> recommend to create a user with sudo permissions, login to DE with root is bad ideas....

here [youtube](https://youtu.be/SDHKjeOGvhQ )i created user and use it. the problem is present.  if i use rdp connection first, after  rustdesk open normally
